deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -52,7 +52,7 @@ spec:
             - "--metrics-address=:22012"
             - "--leader-election"
             - "--leader-election-namespace=$(PDCSI_NAMESPACE)"
-            - "--timeout=250s"
+            - "--timeout=60s"
```

Comment:
I think we should keep the value at the 99% percentile

Reply:
how does a smaller timeout improve recovery if the underlying operation doesn't complete? The main thing I see is we will create an error event more frequently

Reply:
the problem is not operation does not complete. The issue I am seeing (guessing) from logs that the first operation is not created successfully, but it does not return anything until timeout. After that, the second attach is called, and this time operation is created successfully and finished in a few second. I was suspecting it is due to rate limit and operation is dropped. But in this case, I am not sure operation creation returns an error immediately? 

Having a short timeout means we have a chance to create another operation.